x/plans/types/plan.go

```diff
@@ -45,6 +45,10 @@ func (p Plan) ValidatePlan() error {
 		return sdkerrors.Wrap(ErrInvalidPlanAnnualDiscount, "plan's annual discount is invalid (not between 0-100 percent)")
 	}
 
+	if p.GetProjects() == 0 {
```

Comment:
lets do 0 is unlimited

Reply:
done. see https://github.com/lavanet/lava/commit/dd2aede1daae58f2e1605969f08f7679938d3b8e